_maps/shuttles/syndicate/syndicate_cybersun_kansatsu.dmm

```diff
@@ -1,20 +1,4 @@
 //MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```

Comment:
Not sure if this is intended, but it needs to not be in this PR.

Reply:
The branch had some fucky bullshit I tried to do with beacons in it's first iteration and that was a consequence of trying to untangle it.